apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/TablesTest.cs

```diff
@@ -780,5 +780,44 @@ public void GetTable_PartialSchema_ComplexTypes()
             Assert.Equal("gender", table.Schema.Fields[3].Fields[0].Name);
             Assert.Equal("age", table.Schema.Fields[3].Fields[1].Name);
         }
+
+        [Fact]
+        public void GetTableIamPolicy()
+        {
+            var client = BigQueryClient.Create(_fixture.ProjectId);
+            var policy = client.GetTableIamPolicy(
+                _fixture.DatasetId, _fixture.HighScoreTableId, new GetTableIamPolicyOptions { PolicyVersion = 3 });
+
+            Assert.NotNull(policy.ETag);
+        }
+
+        [Fact]
+        public void SetTableIamPolicy()
```

Comment:
Given that this test is mutating the table, perhaps it should create a new table just for that purpose?